utils.py

```diff
@@ -88,11 +108,13 @@ def get_file_contents(filepath, raw_bytes=False, mode='r'):
     else:
         encoding = 'utf-8'
 
-    with python_utils.open_file(filepath, mode, encoding=encoding) as f:
-        return f.read()
+    with python_utils.open_file( # type: ignore[no-untyped-call]
```

Comment:
Why is this ignore needed?

Reply:
This function is not type annotated. As we have specified `disallow-untyped-calls` as true, this means any call of an untyped function will be notified.